RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeedsParameters_cff.py

```diff
@@ -34,7 +34,11 @@
     # H/E towers
     hcalTowers = cms.InputTag("towerMaker"),
     hOverEPtMin = cms.double(0.),
-    
+
+    applySigmaIEtaIEtaCut = cms.bool(True),
+    # sigma_ietaieta
+    maxSigmaIEtaIEtaBarrel = cms.double(0.5),
+    maxSigmaIEtaIEtaEndcaps = cms.double(0.5),    
```

Comment:
Yes, there should be an appropriate file to add the necessary cuts to the HI workflow.
@richard-cms should know where you need to make this customization.